tests/bsim/bluetooth/samples/central_hr_peripheral_hr/src/sample_test.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2023-2024 Nordic Semiconductor ASA
+ * Copyright (c) 2017-2019 Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bs_types.h"
+#include "bs_tracing.h"
+#include "bs_utils.h"
+#include "time_machine.h"
+#include "bstests.h"
+
+#define WAIT_TIME 120 /* Seconds */
+
+#define PASS_THRESHOLD 100 /* packets */
```

Comment:
```suggestion
#define PASS_THRESHOLD 5 /* packets */
```

I don't see any reason to wait for 100 notifications. The behavior should be verifiable with a few notifications